firebase-firestore/src/test/java/com/google/firebase/firestore/UserDataWriterTest.java

```diff
@@ -291,4 +287,8 @@ public void testRejectsJavaArrays() {
   private Object convertValue(FieldValue value) {
     return writer.convertValue(value.getProto());
   }
+
+  private void assertValueType(FieldValue value, Value.ValueTypeCase booleanValue) {
```

Comment:
JUnit tends to put the expected value as the first argument. Consider reversing these to match.

Reply:
Changed this. Note that I have a similar helper for canonical IDs, which I kept the same as the callsites look much cleaner in reverse argument order.